checklist/app.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Application, Context} from 'probot';
+import Webhooks from '@octokit/webhooks';
+import {Octokit} from '@octokit/rest';
+
+import {ChecklistBot} from './src/checklist_bot';
+
+type WebhookPayload = Webhooks.WebhookPayloadPullRequest;
+
+module.exports = (app: Application) => {
+  if (process.env.NODE_ENV !== 'test') {
+    require('dotenv').config();
```

Comment:
I'm pretty sure Probot does this whole thing automatically, you can try removing this condition and see if local dev and tests still work as expected

Reply:
I was under the impression it doesn't cause I had tests acting weird due to dotenv stuff